doc/thingpedia-tutorial-linkedin.md

```diff
@@ -69,8 +80,11 @@ module.exports = class LinkedinDevice extends Tp.BaseDevice {
     /*
     A query function called "get_profile"
     The "get_" prefix indicates this is a query, not an action
+
+    LinkedIn's API used to return more information.
+    Unfortunately, we can only obtain the user's name with the latest API.
```

Comment:
If we keep the LinkedIn example, it's worth killing the query entirely, and leaving only the action. The point of this tutorial is OAuth anyway, and the action is good enough for that.

Reply:
Yup okay, I'll kill query and also remove it from the version on thingpedia-common-devices